force-app/main/default/lwc/apexWireMethodToFunction/__tests__/apexWireMethodToFunction.test.js

```diff
@@ -34,7 +35,7 @@ describe('c-apex-wire-method-to-function', () => {
             getContactListAdapter.emit(mockGetContactList);
             return Promise.resolve().then(() => {
                 const detailEls = element.shadowRoot.querySelectorAll('p');
-                expect(detailEls.length).toBe(2);
+                expect(detailEls.length).toBe(USER_COUNT_RESULT);
```

Comment:
A good example why extracting the expected values from the test data set would make the tests more resilient. Only thing that would need to be changed is the test name, but you can always rename the test to something more generic about it rendering the contact record data.

Reply:
Good point!